src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultFileUriProvider.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Composition;
+using System.IO;
+using Microsoft.VisualStudio.Text;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+{
+    [Shared]
```

Comment:
>[Shared] [](start = 4, length = 8)

Learning opportunity for me... This isn't the default?

Reply:
Alex had a similar question 😄. Just a habit for me: https://github.com/dotnet/aspnetcore-tooling/pull/1602#discussion_r381406922

Will probably break that habit for this PR